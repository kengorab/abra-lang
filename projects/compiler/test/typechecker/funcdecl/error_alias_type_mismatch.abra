func foo(a: Int, b: String): Bool = true

val f: (Int, String) => Float = foo
