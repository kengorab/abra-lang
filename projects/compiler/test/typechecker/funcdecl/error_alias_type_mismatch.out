Error at %FILE_NAME%:3:33
Type mismatch
  |  val f: (Int, String) => Float = foo
                                     ^
Expected: (Int, String) => Float
but instead found: (Int, String) => Bool
